python_modules/dagster-test/dagster_test/toys/external_execution/numbers_example/number_sum.py

```diff
@@ -0,0 +1,13 @@
+from dagster_external import init_dagster_external
+
+from .util import compute_data_version, load_asset_value, store_asset_value
+
+context = init_dagster_external()
+storage_root = context.get_extra("storage_root")
+number_y = load_asset_value("number_y", storage_root)
+number_x = load_asset_value("number_x", storage_root)
```

Comment:
I think being FS storage oriented wont translate well to the other substrates, its not too bad for docker on same node but gets more difficult past that

Reply:
What would you recommend instead?